### PR TITLE
Update docs to move RAPIDS Shuffle out of beta

### PR DESCRIPTION
**Pull request by sameerz**
Signed-off-by: Sameer Raheja <sraheja@nvidia.com>

Update docs to remove beta designation from RAPIDS Shuffle

### CODE REVIEW

**Review by abellina — APPROVED** [review]

**Comment by abellina** [comment]
build

File: docs/download.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ New functionality for this release includes:
 * Support for the `concat_ws` operator
 
 Performance improvements for this release include: 
+* Moving RAPIDS Shuffle out of beta
```
Comment (abellina) [inline docs/download.md:75]: Sorry, not entirely sure on this one. Should it be under "Performance improvements"?

**Comment by sameerz** [comment]
build

**Comment by sameerz** [comment]
Replaced by PR #2665

**Pull request closed without merging**